core/src/main/java/org/apache/accumulo/core/client/sample/Sampler.java

```diff
@@ -56,7 +56,7 @@
   /**
    * @param k
    *          A key that was written to a rfile.
-   * @return True if the key (and its associtated value) should be stored in the rfile's sample.
+   * @return True if the key (and its associated value) should be stored in the rfiles sample.
```

Comment:
The apostrophe for rfile is showing possession here so should be "rfile's"

Reply:
Fixed